cpp/CMakeLists.txt

```diff
@@ -470,8 +470,14 @@ target_link_libraries(cudf
                   rmm::rmm)
 
 if(CUDA_STATIC_RUNTIME)
```

Comment:
Just a note but in the future we should probably get rid of the `CUDA_STATIC_RUNTIME` option and just use `CMAKE_CUDA_RUNTIME_LIBRARY` instead to control things.

Reply:
I have created feature request https://github.com/rapidsai/cudf/issues/7888 to track that.